features/step_definitions/animal_steps.rb

```diff
@@ -1 +1,19 @@
-#Delete this comment, here is where you should write your step defs
+Given(/^a 4 year old cat named Lucy$/) do
+  @animal = Animal.new('Lucy', 'cat', 4)
+end
+
+Then(/^I should receive Lucy when I ask for its name$/) do
+  expect(@animal.name).to eq('Lucy')
+end
+
+Then(/^I should receive cat when I ask for its type$/) do
+  expect(@animal.type).to eq('cat')
+end
+
+Then(/^I should receive 4 when I ask for its age$/) do
+  expect(@animal.age).to be 4
+end
+
+Then(/^I should receive true when I ask if its older$/) do
+  expect(@animal.old?).to be true 
+end
```

Comment:
I recommend using regular expression capture groups or Cucumber's `$` syntax for your step definitions so you can change values in your steps without having to update your step definitions (see https://github.com/cucumber/cucumber/wiki/Step-Definitions).

For example:
```ruby
Given "a $age year old $animal named $name" do |age, animal, name|
  @animal = Animal.new(name, animal, age)
end
```